p3/plot/backend/pgfplots.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, df, eff=None, size=None, stream=None, **kwargs):
             app_df = app_df.sort_values(by=[eff_column], ascending=False)
             yvalues = app_df[eff_column]
             supported_platforms = list(app_df["platform"])
-            xvalues = np.arange(1, len(supported_platforms) + 1)
+            xvalues = [x for x in range(1, len(supported_platforms) + 1)]
```

Comment:
Just to chime in, `range` looks better but not sure but since it returns a generator (and not a list) sometimes behavior might change a little, so if you try this suggestion please make sure that the plotting still works!